docs/getting-started/installation.md

```diff
@@ -43,40 +49,77 @@ pip install -U prefect
 ```
 </div>
 
-To install a specific version, specify the version number, like this:
+To install a specific version, specify the version number like this:
 
 <div class="terminal">
 ```bash
-pip install -U "prefect==2.16.2"
+pip install -U "prefect==2.17.1"
 ```
 </div>
 
-See available release versions in the [Release Notes](https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md).
+See available release versions in the [Prefect Release Notes](https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md).
+
+### Installing the bleeding edge
```

Comment:
Who is the intended audience for this section? Does it need to be as prominent as it is?

Reply:
Hmm, was present before. I think the idea is probably that here you're specifying a specific version number, so to learn about specific versions, see the release notes.

Reply:
Ah sorry, I was referring to the "bleeding edge" section

Reply:
Oh yeah, that was present before, I think I tried to de-center it a bit - will look at how to do that more.

Reply:
Moved this lower in the page and link out contributing.